postgres/client.sls

```diff
@@ -23,6 +23,9 @@ postgresql-client-libs:
     - refresh: True
     - require:
       - pkgrepo: postgresql-repo
+       {% if grains.os_family == 'Suse' %}
```

Comment:
I meant, no need for `{% if grains... %}` statements and `pkgrepo` requisite above as well 😃

Reply:
no problem. check now.